entrypoint.sh

```diff
@@ -3,10 +3,17 @@
 export RUNNER_ALLOW_RUNASROOT=1
 export PATH=$PATH:/actions-runner
 
+# Un-export these, so that they must be passed explicitly to the environment of
+# any command that needs them.  This may help prevent leaks.
+export -n ACCESS_TOKEN
+export -n RUNNER_TOKEN
```

Comment:
this works in theory but not via docker, probably because the cmd is not a separate process (retains the same pid)?

I built this and ran it with 

```bash
docker run \
  -e REPO_URL="https://github.com/OctoKode/test1" \
  -e RUNNER_NAME="foo-runner" \
  -e RUNNER_TOKEN="AAGW...snip....WF6Q" \
  wut
```

then docker exec'd into the container:

```bash
root@f5e6653bcb76:/actions-runner# env | grep -i token
RUNNER_TOKEN=AAGW...snip....WF6Q
```

Reply:
Actually looks like it doesnt work on my master branch either 🤔